### PR TITLE
Multi root adoption

### PR DESCRIPTION
**Pull request by aeschli**

### CODE REVIEW

**Comment by fbricon** [comment]
@aeschli can you sign-off your last commit please?

File: package.json [hunk 1]
```diff
@@ -58,15 +58,17 @@
             "null"
```
Comment (fbricon) [inline package.json]: should we add the `multi-root ready` keyword as [recommended](https://github.com/Microsoft/vscode/wiki/Extension-Authoring:-Adopting-Multi-Root-Workspace-APIs) or is it too early?

**Comment by aeschli** [comment]
image

**Comment by aeschli** [comment]
Noy sure why this doesn't work.
image

**Comment by fbricon** [comment]
@gorkem ^^

**Comment by fbricon** [comment]
rebased as https://github.com/redhat-developer/vscode-java/commit/5a0ef763fb52a3a886f6b24d65388d7643e827dc

**Pull request closed without merging**